tutorials/workflow/python/external-system-interaction/README.md

```diff
@@ -0,0 +1,126 @@
+# External Events
+
+This tutorial demonstrates how to author a workflow where the workflow will wait until it receives an external event. This pattern is often applied in workflows that require an approval step. For more information about the external system interaction pattern see the [Dapr docs](https://docs.dapr.io/developing-applications/building-blocks/workflow/workflow-patterns/#external-system-interaction).
+
+## Inspect the code
+
+Open the `external_events_workflow.py` file in the `tutorials/workflow/python/external-system-interaction/external_events` folder. This file contains the definition for the workflow. It is an order workflow that requests an external approval if the order has a total price greater than 250 dollars.
+
+```mermaid
+graph LR
+   SW((Start
+   Workflow))
+   IF1{Is TotalPrice
+    > 250?}
+   IF2{Is Order Approved
+   or TotalPrice < 250?}
+   WAIT[Wait for
+   approval event]
+   EX{Event
+   received?}
+   PO[Process Order]
+   SN[Send Notification]
+   EW((End
+   Workflow))
+   SW --> IF1
+   IF1 -->|Yes| WAIT
+   IF1 -->|No| IF2
+   EX -->|Yes| IF2
+   EX -->|No| SN
+   WAIT --> EX
+   IF2 -->|Yes| PO
+   PO --> SN
+   IF2 -->|No| SN
+   SN --> EW
+```
+
+## Run the tutorial
+
+1. Use a terminal to navigate to the `tutorials/workflow/python/external-system-interaction/external_events` folder.
+2. Install the dependencies using pip:
+
+    ```bash
+    pip3 install -r requirements.txt
+    ```
+
+3. Navigate one level back to the `external-system-interaction` folder and use the Dapr CLI to run the Dapr Multi-App run file
+
+    <!-- STEP
+    name: Run multi app run template
+    expected_stdout_lines:
+    - 'Started Dapr with app id "externalevents"'
+    expected_stderr_lines:
+    working_dir: .
+    output_match_mode: substring
+    background: true
+    sleep: 15
+    timeout_seconds: 30
+    -->
+    ```bash
+    dapr run -f .
+    ```
+    <!-- END_STEP -->
+
+4. Use the POST request in the [`externalevents.http`](./externalevents.http) file to start the workflow, or use this cURL command:
+
+    ```bash
+    curl -i --request POST \
+    --url http://localhost:5258/start \
+    --header 'content-type: application/json' \
+    --data '{"id": "b7dd836b-e913-4446-9912-d400befebec5","description": "Rubber ducks","quantity": 100,"totalPrice": 500}'
+    ```
+
+    The input for the workflow is an `Order` object:
+
+    ```json
+    {
+        "id": "{{orderId}}",
+        "description": "Rubber ducks",
+        "quantity": 100,
+        "totalPrice": 500
+    }
+    ```
+
+5. Use the GET request in the [`externalevents.http`](./externalevents.http) file to get the status of the workflow, or use this cURL command:
+
+    ```bash
+    curl --request GET --url http://localhost:3558/v1.0/workflows/dapr/b7dd836b-e913-4446-9912-d400befebec5
```

Comment:
```suggestion
    curl --request GET --url http://localhost:3558/v1.0/workflows/dapr/<INSTANCEID>
    ```

    Where `<INSTANCEID>` is the workflow instance ID you received in the `instance_id` 
     property in the previous step.
```

Reply:
this one didnt look resolved @marcduiker . Reason you didn't want to use the generic here?